/***************************************************************
Get Device Configuration Message proto file
- command/response definition
****************************************************************/
syntax = "proto3";
option go_package = "pooltester3/ned";

package ned;

message GetDeviceConfigurationRequestPayload{} // No fields, juste used for identification

message GetDeviceConfigurationResponsePayload {
    string core_bssid = 1; // Core BSSID only if the current bus is wifi, otherwise an empty string
    bool is_find_me_active = 2; // True if find me is active, false otherwise
}
